### DISCUSSION TITLE
Command + ] for alternative suggestions => Not suited for Spanish keyboards

### DISCUSSION THREAD

**Discussion by imayolas** [community/community#39176] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Context: Copilot in Visual Studio Code

Spanish keyboards have the `]` character as the third option of the `+` key; meaning that in order to trigger it the user must hit: `Option + +`.

What this implies is that a user with a Spanish keyboard can't switch Copilot's suggestions by hitting `Command ]` as what he's actually sending is `Command +`.

Similarly, the combo of `Command Option +`, doesn't trigger any suggested alternatives.

Is there a way to set up a different command to trigger Copilot's alternative suggestions? If not, it would be a great addition!

### Screenshot of the key
![keyboard](https://user-images.githubusercontent.com/6229037/202983393-36648726-e3c3-430a-97ca-4f284eda57a6.jpg)

**Comment by iamcjwilliams** [comment]
@imayolas Thank you for your suggestion, our copilot will be notified!

**Comment by andreas83** [comment]
same issue here. (german layout under linux)

would be nice if i could configure the keyboard binding myself.

**Comment by villesau** [comment]
Looks like very similar problem what I reported about Fin & Swe keyboards: https://github.com/community/community/discussions/41353

Seems like a real issue for a big part of the European users.
